linode/manager
# change: [M3-7683] - Keep Create & Upload Image Pages' Error Notification Position Consistent

## Description 📝
Keeping the Create & Upload Image Pages' error notification position consistent with other tabs like NodeBalancers.

## Changes  🔄
 - For restricted users:
    - The position of error notification has been moved from inside the box to outside, which is now consistent with other tabs like NodeBalancers.
    - Used getRestrictedResourceText utility to show a Notice / error notification message.

## Target release date 🗓️


## Preview 📷

Create Image Page:
| Before  | After   |
| ------- | ------- |
|![11_39_36](https://github.com/user-attachments/assets/46f72d08-27b7-4b9d-8714-c50a8998e0d0)|![11_33_02](https://github.com/user-attachments/assets/50567c27-690f-4181-b5e6-cd4fb561ee8b)|

Upload Image Page:
| Before  | After   |
| ------- | ------- |
|![11_39_56](https://github.com/user-attachments/assets/b2708b81-5e1c-4d62-9a32-5d10466b5e3b)|![11_33_23](https://github.com/user-attachments/assets/e08cf70d-d56e-440d-a661-f53265aa6b4e)|


## How to test 🧪

### Prerequisites
(How to setup test environment)
- For create page:
  - Log in with a restricted user account (read-only access) and go to the Create Images Page via images/create/disk route directly from you browser.
- For upload page: 
  - Log in with a restricted user account (read-only access) and go to the Upload Images Page via images/create/upload route directly from you browser.

### Reproduction steps
- Observe as restricted user, error notification shows out of the box (not inside as before) and you cannot create/upload Images.


### Verification steps
- As a restricted user, compare Create and Upload Images Page (routes: images/create/disk & images/create/upload respectively) with Create NodeBalancers Page (route: nodebalancers/create) to ensure consistency.


## As an Author I have considered 🤔

*Check all that apply*
